src/script.lua

```diff
@@ -1892,7 +1901,7 @@ function dataIsOfType(data, target_type, caller)
 		return validatePlayerID(as_num, caller)
 	elseif target_type == "number" then
 		return as_num ~= nil, as_num, not as_num and ((data or "nil") .. " is not a valid number")
-	elseif target_type == "bool" then
+	elseif --[[todo: bool is not correct ]] target_type == "bool" or target_type == "boolean" then
```

Comment:
@CodeLeopard What is this for? Can `target_type == "bool` not be removed?

Reply:
I didn't know if there might have been more bools lurking in places. Can be removed I guess.